src/index.js

```diff
@@ -0,0 +1,195 @@
+const cityUpdate = () => {
+  const newCityText = state.cityTextField.value;
+  state.headerCity.textContent = newCityText;
+};
+
+const cityDefault = "Boston";
+
+const cityReset = () => {
+  state.cityTextField.value = cityDefault;
+  state.headerCity.textContent = state.cityTextField.value;
+};
+
+const landscapes = {
+  coldLandscape: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+  coolLandscape: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+  warmLandscape: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+  hotLandscape: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+};
+
+const skyOptions = {
```

Comment:
For the sky values, a dictionary works well since we can look up the options directly. We could even iterate over the dictionary keys (or use a list similar to that described for the landscape) in order to generate the lookup dictionary, and populate the options in the select. This approach can be useful if the options are to be loaded from a backend database!

However, prefer to use the values in the select element as the keys directly so that we can look up the sky value using the selected sky value rather than needing an additional function to translate from the selected sky value to the sky key (`skyOptionsUpdate`).